editor/UserInterface/Vector2Config.cs

```diff
@@ -0,0 +1,140 @@
+﻿using OpenTK;
+using StorybrewEditor.Graphics;
+using StorybrewEditor.Graphics.Drawables;
+using StorybrewEditor.UserInterface.Skinning.Styles;
+using StorybrewEditor.Util;
+using System;
+
+namespace StorybrewEditor.UserInterface
+{
+    public class Vector2Config : Widget, Field
+    {
+        private LinearLayout layout;
+        private Textbox xTextbox;
+        private Textbox yTextbox;
+
+        public override Vector2 MinSize => new Vector2(layout.MinSize.X, layout.MinSize.Y);
+        public override Vector2 MaxSize => Vector2.Zero;
+        public override Vector2 PreferredSize => new Vector2(layout.PreferredSize.X, layout.PreferredSize.Y);
+
+        private Vector2 value;
+        public Vector2 Value
+        {
+            get { return value; }
+            set
+            {
+                if (this.value == value) return;
+                this.value = value;
+
+                updateWidgets();
+                OnValueChanged?.Invoke(this, EventArgs.Empty);
+            }
+        }
+
+        public object FieldValue
+        {
+            get { return Value; }
+            set { Value = (Vector2)value; }
+        }
+
+        public event EventHandler OnValueChanged;
+        public event EventHandler OnValueCommited;
+
+        public Vector2Config(WidgetManager manager) : base(manager)
+        {
+            Add(layout = new LinearLayout(manager)
+            {
+                StyleName = "condensed",
+                FitChildren = true,
+                Children = new Widget[]
+                {
+                    new Label(Manager)
+                    {
+                        StyleName = "small",
+                        Text = "X"
+                    },
+                    xTextbox = new Textbox(manager)
+                    {
+                        EnterCommits = true,
+                    },
+                    new Label(Manager)
+                    {
+                        StyleName = "small",
+                        Text = "Y"
+                    },
+                    yTextbox = new Textbox(manager)
+                    {
+                        EnterCommits = true,
+                    },
+                },
+            });
+            updateWidgets();
+
+            xTextbox.OnValueCommited += xTextbox_OnValueCommited;
+            yTextbox.OnValueCommited += yTextbox_OnValueCommited;
+        }
+
+        private void xTextbox_OnValueCommited(object sender, EventArgs e)
+        {
+            var xCommit = xTextbox.Value;
+
+            float X;
```

Comment:
I wanted to keep in consistency with Vector2's convention of capitalizing the X and Y values for their components. But changed, nevertheless.